invenio/modules/knowledge/models.py

```diff
@@ -1,7 +1,7 @@
 # -*- coding: utf-8 -*-
 ##
 ## This file is part of Invenio.
-## Copyright (C) 2011, 2012, 2014 CERN.
+## Copyright (C) 2011, 2012, 2013, 2014, 2015 CERN.
```

Comment:
Tiny detail…the lack of 2013 just means that the file was not edited in 2013, so it's ok that it is not there.